gui/src/app/view/psbt.rs

```diff
@@ -729,12 +739,12 @@ pub fn inputs_and_outputs_view<'a>(
 }
 
 fn input_view<'a>(
-    coin: &'a Coin,
+    outpoint: &'a OutPoint,
+    coin: Option<&'a Coin>,
     labels: &'a HashMap<String, String>,
     labels_editing: &'a HashMap<String, form::Value<String>>,
 ) -> Element<'a, Message> {
-    let outpoint = coin.outpoint.to_string();
-    let addr = coin.address.to_string();
+    let outpoint = outpoint.to_string();
     Column::new()
         .width(Length::Fill)
         .push(
```

Comment:
Can the labelling be turned off for an `outpoint` with no associated `coin`? I'm still given the option to add a label, and then any value I enter disappears after saving.